app/assets/javascripts/discourse/app/templates/topic.hbs

```diff
@@ -348,22 +314,58 @@
                 </div>
               {{/if}}
 
-              {{plugin-outlet name="topic-above-suggested" args=(hash model=model)}}
-              <div class="{{if model.relatedMessages.length "related-messages-wrapper"}} {{if model.suggestedTopics.length "suggested-topics-wrapper"}}">
-                {{#if model.relatedMessages.length}}
-                  {{related-messages topic=model}}
-                {{/if}}
-                {{#if model.suggestedTopics.length}}
-                  {{suggested-topics topic=model}}
-                {{/if}}
-              </div>
             {{/if}}
           {{/conditional-loading-spinner}}
 
         </section>
       </div>
 
     </div>
+    {{#if loadedAllPosts}}
```

Comment:
note that topic.hbs is overridden by multiple themes, so the merge of this kind of PR should be followed up very cautiously